src/main/java/com/redhat/cloud/policies/app/health/ScheduledStatusProducer.java

```diff
@@ -44,15 +45,21 @@ public class ScheduledStatusProducer {
     @RestClient
     PolicyEngine engine;
 
-    //  // Quarkus only activates this after the first REST-call to any method in this class
-    @Gauge(name = "status_isDegraded", unit = MetricUnits.NONE, absolute = true,
-            description = "Returns 0 if good, value > 0 for number of entries in the status message")
-    int isDegraded() {
```

Comment:
A gauge needs to be "manually" updated by code. This accessor is only for the metrics subsystem to retrieve the value at the moment the /metrics endpoint is called.

Reply:
Hm. I don't really understand how the status_isDegraded metric is set to 1.0 in our code :/

Reply:
We are changing the status in AdminService and this code is called in the tests. But neither the AdminService nor the Test has changed so that it doesn't set the isDegraded from 0.0 to 1.0